gcb/build_cert_manager.yaml

```diff
@@ -15,7 +15,7 @@ steps:
   args: ['fetch', '--unshallow']
 
 ## Build release artifacts and push to a bucket
-- name: 'eu.gcr.io/jetstack-build-infra-images/make-dind:20230406-0ef4440-bullseye'
+- name: 'europe-west1-docker.pkg.dev/cert-manager-tests-trusted/cert-manager-infra-images/make-dind:20240422-6b43e85-bookworm'
```

Comment:
I [searched the cert-manager org for `make-dind`](https://github.com/search?q=org%3Acert-manager+make-dind&type=code) and found this image reference in the [testing repo](https://github.com/cert-manager/testing/blob/23e6a79cb2e053a08994070c07e3a9befe1907fd/config/prowgen/pkg/globals.go#L22).